packages/server/test/migration/0.1.20-fix-client-schema.test.ts

```diff
@@ -60,8 +60,7 @@ describe('0.1.20 - Fix Client Schema', () => {
     const id = uuid()
```

Comment:
I understand why you did those changes, but since this test should be done on a DB that matches with the server v0.1.20, I think we should instead add a mechanism that easily allows us to roll back the DB to a certain version before running migration tests. This way, we won't have to make those changes and the tests would be more accurate. What do you think?

Reply:
Agreed. I'll make changes to how migration tests are done in #299. I think it's ok to merge this for now